### PR TITLE
Changes chapter 8

### PR DESCRIPTION
**Pull request by AlessandroMiola**
Hi @mrdbourke,
thank you for the amazing course and chapter.

Here are a couple of typos' fixes: 
- in the pseudocode of equation 4, _layer norm_ should be applied first with respect to the _linear layer_;
- _dropout_ should be a _float_ rather than an _int_;
- _weight_decay_'s default might be set to 0.03 rather than to 0.1 given that we're kind of following the setup of _ViT-B/16_ trained on _ImageNet-21k_ rather than on _JFT-300M_.

### CODE REVIEW

**Comment by mrdbourke** [comment]
Hi @AlessandroMiola,

Thank you for this!

I agree with all the changes so far.

But to make things a little clearer, do you might add a small line to each hyperparameter change?

Eg.

Current:

```python
# Old
...(weight_decay=0.03) # from the ViT paper section 4.1 (Training & Fine-tuning)
```

New (add Table 3):

```python
# Old
...(weight_decay=0.03) # from the ViT paper section 4.1 (Training & Fine-tuning) and Table 3 for ViT-B/16 ImageNet21k
```

Hmm...

Now that I think about it, we're more so producing the ImageNet1k version (I'd prefer it this way, start simpler first).

Perhaps you'd like to update the parameters to reflect this version:

Screenshot 2023-01-19 at 11 55 00 am

If you think the notebook could be updated in any other way to make this more explicit, please let me know - I'd be happy to merge this as a PR.

**Comment by AlessandroMiola** [comment]
Just updated it. 
Given that the notebook is already very clear in the _pre-changes status_ (with the default `weight_decay=0.1` which references section 4.1 _Training & Fine-tuning_) I can also revert to that status. 
 
I did add the change on weight decay default value just because it seemed to me that the video was referencing another model compared to the general setting of the paper (which mostly references the big models trained on JFT-300M dataset).

**Comment by mrdbourke** [comment]
Thank you so much for this! Merging!
